api/system-authorization/read

```diff
@@ -142,5 +142,6 @@ print encode_json({
     "OtpStatus" => $OtpStatus,
     "OtpCockpit" => $OtpCockpit,
     "OtpSshd" => $OtpSshd,
+    "OtpR2WOath" => (-f "/var/lib/nethserver/openvpn-certificate-otp/".getpwuid($<)) ? "enabled":"disabled",
```

Comment:
Shouldn't be `$username` instead of `getpwuid`?

Reply:
at the top of the file we remove the @domainName, and I need the username@domainName